linkerd/tracing/Cargo.toml

```diff
@@ -15,15 +15,16 @@ bytes = "1"
 html-escape = "0.2.5"
 http = "0.2"
 hyper = { version = "0.14.2", features = ["http1"] }
+linkerd-access-log = { path = "../access-log" }
```

Comment:
@hawkw Do you have any thoughts on how we can avoid having `tracing` depend on `access-log`? My concern is that now the tracing dependency pulls in a bunch of HTTP, etc specific dependencies that shouldn't be needed by such a core dependency like this.

Reply:
Perhaps the tracing crate should expose `AccessLogWriter` and we should pull that in from the access-log crate so that the dependencies are inverted?

Reply:
I'm just going to give this a try and push these changes to the branch.